src/basis/utils/json-parser.js

```diff
@@ -223,15 +223,15 @@ var JsonParser = (function () {
             value: function _testKeyword() {
                 var _this = this;
 
-                var matched = Object.keys(keywordsTokens).find(function (name) {
+                var matched = Object.keys(keywordsTokens).filter(function (name) {
```

Comment:
it's better to contribute back to https://github.com/vtrushin/json-to-ast
and `for..in` with no substr on every keywordsTokens key is better